pyomo/solvers/plugins/solvers/GAMS.py

```diff
@@ -733,11 +740,19 @@ def solve(self, *args, **kwds):
 
         exe = self.executable()
         command = [exe, output_filename, 'o=' + lst_filename]
-        if not tee:
+        if tee and not logfile:
+            pass  # default behaviour of gams is to print to console
```

Comment:
I think we can fix Windows tests by changing this path from an implicit "`lo=1`" to an explicit "`lo=3`".  From the GAMS documentation (https://www.gams.com/latest/docs/UG_GamsCall.html):

Value | Meaning
-- | --
0 | No log output
1 | Log output to screen (console)
2 | Log output to logfile
3 | Log output to standard output
4 | Log output to logfile and standard output

Note that on *nix, the console and stdout are usually synonymous; however, on Windows they are not.  Making the output explicitly go to stdout should get redirection working.

Reply:
Thanks!